__tests__/app.test.js

```diff
@@ -80,3 +80,47 @@ describe("GET /api/articles/:articleid", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:articleid", () => {
+  const articleid = 2;
```

Comment:
Since you are reusing this variable name in a lower scope, i would put it inside of the test blocks that use it to avoid confusion